Remnant: Bellows-Class Afterburner (#4652)
endless-sky/endless-sky: data/remnant/remnant outfits.txt

# Engines:

outfit "Bellows-Class Afterburner"
	category "Engines"
	"cost" 640000
	thumbnail "outfit/remnant afterburner"
	"mass" 13
	"outfit space" -11
	"engine capacity" -11
	"afterburner thrust" 153.0
	"afterburner fuel" 2.0
	"afterburner energy" 6.0
	"afterburner heat" 8.0
	"afterburner effect" "remnant afterburner"

> 	description "The Remnant live in an environment where the raw materials for fuel are relatively common on the winds of the nebula, and with ships that incorporate significant built-in ramscoop technology. As such, while they are reluctant to waste fuel, neither are they afraid of using it when needed. Far more important is the ability for their ships to quickly and efficiently enter or exit combat as needed. At the end of the day, it is continued life that is paramount."

Reviewer: ```suggestion
	description "The Remnant live in an environment where the raw materials for fuel are relatively common on the winds of the nebula. As such, their ships incorporate significant built-in ramscoop technology, and while they are reluctant to waste fuel, they aren't afraid of using it when needed. Far more important is the ability for their ships to quickly and efficiently enter or exit combat as needed. At the end of the day, it is continued life that is paramount."
```

> 	description "	The bellows-class afterburner embodies this philosophy by providing captains with a tool that allows them to make great leaps of acceleration, typically during evasive maneuvers or to catch up with a fleeing target. Due to their enormous fuel usage, it is rare for any ship to use these for more than a few seconds at a time."

Reviewer: Given that this thing guzzles 120 fuel per second of usage, I'd hardly even say that they are rarely used for more than a few seconds; after a few seconds, I'm already out of fuel!
```suggestion
	description "	The Bellows-class afterburner embodies this philosophy by providing captains with a tool that allows them to make great leaps of acceleration, typically during evasive maneuvers or to catch up with a fleeing target. Due to their enormous fuel usage, all but the most specialized of ships have enough fuel to use these for more than a few seconds at a time before completely running out of fuel."
```

Reviewer 2: "all but the most specialized of ships have enough fuel"? I think you meant "only the most specialized of ships have enough fuel". And then the extra bit is not needed.

Author: Thanks, I actually caught that at the time, and it should already read "none but the most specialized of ships..."

Reviewer: Oops. I had like three different things typed out as to how to change this up and I guess I accidentally smashed them together. 

“All but the most specialized ships run out of fuel in a few seconds” vs “only the most specialized ships can use this for more than a few seconds”.

> 	description "	The bellows-class afterburner embodies this philosophy by providing captains with a tool that allows them to make great leaps of acceleration, typically during evasive manouevers or to catch up with a fleeing target. Due to their enormous fuel usage, it is rare for any ship to use these for more than a few seconds at a time."

Reviewer 3: ```suggestion
	description "	The bellows-class afterburner embodies this philosophy by providing captains with a tool that allows them to make great leaps of acceleration, typically during evasive maneuvers or to catch up with a fleeing target. Due to their enormous fuel usage, it is rare for any ship to use these for more than a few seconds at a time."
```

> 	description "	The Bellows-class afterburner embodies this philosophy by providing captains with a tool that allows them to make great leaps of acceleration, typically during evasive maneuvers or to catch up with a fleeing target. Due to their enormous fuel usage, none but the most specialized of ships have enough fuel to use these for more than a few seconds at a time before completely running out of fuel."

Reviewer: "None but" is an odd wording. Just go with "only".
```suggestion
	description "	The Bellows-class afterburner embodies this philosophy by providing captains with a tool that allows them to make great leaps of acceleration, typically during evasive maneuvers or to catch up with a fleeing target. Due to their enormous fuel usage, only the most specialized of ships have enough fuel to use these for more than a few seconds at a time before completely running out of fuel."
```
